src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -1147,11 +1146,8 @@ def check_read_nwb(self, nwbfile_path: str):
             data_from_extractor = imaging_extractor.get_video()
             assert_array_equal(two_photon_series.data[:], data_from_extractor.transpose(0, 2, 1))
 
-            assert two_photon_series.description == json.dumps(self.interface.image_metadata)
-
             optical_channels = nwbfile.imaging_planes[self.imaging_plane_name].optical_channel
             optical_channel_names = [channel.name for channel in optical_channels]
-            assert self.interface_kwargs["channel_name"] in optical_channel_names
```

Comment:
The optical channels are no longer named like the channel so this test does not make sense.